packages/devtools_app/lib/src/shared/framework_controller.dart

```diff
@@ -36,13 +36,8 @@ class FrameworkController {
   Stream<ConnectVmEvent> get onConnectVmEvent => _connectVmController.stream;
 
   /// Notify the controller of a connect to VM event.
-  void notifyConnectToVmEvent(Uri serviceProtocolUri, {bool notify = false}) {
```

Comment:
please revert these changes. ConnectVmEvent's fields are actually used here, but it looks like these are getting flagged as unused because we are mistakenly using dynamic execution:
https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/framework/initializer.dart#L107-L112

Can you please add the type `ConnectVmEvent` to the parameter event in this method? and then revert these changes. thanks

Reply:
Wasn't expecting that 😅. Done